extensions/scarb-cairo-run/src/main.rs

```diff
@@ -49,7 +50,10 @@ struct Args {
 }
 
 fn main() -> Result<()> {
-    let ui = Ui::new(Verbosity::default(), OutputFormat::Text);
+    let verbosity = env::var_os("SCARB_UI_VERBOSITY")
+        .and_then(|level| scarb_ui::Verbosity::from_str(&level.to_string_lossy()).ok())
+        .unwrap_or_default();
+    let ui = Ui::new(verbosity, OutputFormat::Text);
```

Comment:
IMHO, `scarb_ui` should provide ready-to-use clap arguments and the UI should be able to be constructed from them in a one-liner. Too much logic becomes smeared throughout different components.